mpas_analysis/config.default

```diff
@@ -731,10 +731,10 @@ colorbarLevelsDifference = [-5, -3, -2, -1, -0.1, 0, 0.1, 1, 2, 3, 5]
 contourLevelsDifference = np.arange(-5., 6., 2.)
 
 [streamfunctionMOCIndoPacific]
-## options related to plotting the Global MOC
+## options related to plotting the IndoPacific MOC
 
 # Minium value of latitudinal extent for MOC streamfunction plot
-latBinMin = -80.
+latBinMin = -35.
```

Comment:
Below, please change the colormapIndicesResult to `colormapIndicesResult = [0, 20, 40, 80, 110, 140, 170, 200, 230, 242, 255]` (same as the other MOC streamfunctions). Thanks!